exercises/practice/intergalactic-transmission/test_intergalactic_transmission.c

```diff
@@ -0,0 +1,347 @@
+#include "intergalactic_transmission.h"
+
+#include "test-framework/unity.h"
+
+#include <stdint.h>
+
+#define BUFFER_SIZE 100
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+#define WRONG_PARITY -1
```

Comment:
I think this already comes from `intergalactic_transmission.h`

Reply:
Good catch! Removed.